File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        numSet = set(nums)
        index = 0
        for val in numSet:
            if val - 1 not in numSet:
                length = 1

                while (val + length) in numSet:
                    length += 1
                index = max(index, length)

        return index
            

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.java
class Solution {
    public int[] maxSlidingWindow(int[] nums, int k) {
        int[] results = new int[nums.length - k + 1];

        for(int left = 0; left <= nums.length - k; left++) {
            int maxI = nums[left];
            for(int right = left; right < left + k; right++) {
                maxI = Math.max(maxI, nums[right]);
            }
            results[left] = maxI;
        }

        return results;
    }
}
